contracts/assetsPool/AssetPoolUpgradeable.sol

```diff
@@ -0,0 +1,154 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+import "../interfaces/LiquidityPoolInterfaces.sol";
+import "./proxy/Proxy.sol";
+import "./token/CRToken.sol";
+import "@openzeppelin/contracts-upgradeable/security/ReentrancyGuardUpgradeable.sol";
+import "@uniswap/lib/contracts/libraries/TransferHelper.sol";
+
+/**
+ * @author CRUIZE.
+ * @title Cruize AssetPool.
+ */
+contract AssetPoolUpgradeable is
+    Proxy,
+    OwnableUpgradeable,
+    ReentrancyGuardUpgradeable
+{
+    mapping(address => address) public lpTokens;
+    // change WETH address when you use different test netwrok CURRENT WETH KOVEN network address.
```

Comment:
Hey what are we trying to say in this comment?
Needs to be phrased better 😅.

Reply:
lol, 😂  that's called a quick bet.